request/parser.go

```diff
@@ -14,6 +14,13 @@ import (
 	"github.com/buildwithgrove/path/relayer"
 )
 
+const (
+	// TODO_DISCUSS: should we create a common errors package for all errors?
```

Comment:
Yes, let's create an `request/errors.go` in this PR.

Reply:
Done.